website/pages/submit-project.html

```
@@ -1,165 +1,286 @@
+            </div>
+
+            <div class="form-group">
+                <label>Challenges Faced</label>
+                <textarea rows="3" placeholder="Describe challenges..."></textarea>
+            </div>
+        </div>
+
+        <!-- Terms -->
+        <div class="form-section">
+            <div class="checkbox-label">
+                <input type="checkbox">
```

HTMLHint (1.9.0)

[warning] The empty tag : [ input ] must be self closed.

(tag-self-close)